src/main/java/uk/gov/hmcts/reform/civil/service/InitiateGeneralApplicationService.java

```diff
@@ -331,6 +332,20 @@ public boolean respondentAssigned(CaseData caseData) {
         String caseId = caseData.getCcdCaseReference().toString();
         CaseAssignmentUserRolesResource userRoles = getUserRolesOnCase(caseId);
         List<String> respondentCaseRoles = getRespondentCaseRoles(caseData);
+
+        if (featureToggleService.isGaForLipsEnabled() && (caseData.isRespondent1LiP() || caseData.isRespondent2LiP()
+            || caseData.isApplicantNotRepresented())) {
```

Comment:
assuming this condition will allow  to check for  LiP vs LR , is this expected , ideally should be checked for lip vs lip and lr vs lip